package.json

```diff
@@ -23,14 +23,17 @@
     "test": "tests"
   },
   "scripts": {
-    "test": "npm run lint && mocha --recursive tests/*.spec.js -R spec && node tests/e2e.js",
+    "test": "npm run lint && npm run test-node && npm run test-electron",
+    "test-node": "mocha --recursive tests/*.spec.js -R spec && node tests/e2e.js",
+    "test-electron": "electron-mocha --recursive tests/*.spec.js -R spec",
```

Comment:
They're currently not run with mocha, but I guess we could port them? /cc @jviotti